Release-Notes/Release3.md

```diff
@@ -124,7 +124,11 @@ docker-compose is then no longer available and docker compose must be used inste
 
 ## Resolved Issues
 
-* certificate rotation in k8s clusters
+* Certificates in k8s clusters can expire (typically after one year).
```

Comment:
please without ( )

* Certificates in k8s clusters are subject to expiration - typically after one year.

Reply:
OK, reworded.